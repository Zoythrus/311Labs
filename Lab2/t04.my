we cockel, cockel evy evy cockel meanding cockel saunted 

cockel cockel cockel cockel, evy evy meanding saunted we 
