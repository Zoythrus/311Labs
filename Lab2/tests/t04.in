er
The King started making all the arrangements, assisted by his secretary, the crab, and when all was ready and all the heads were lined up in the village square, he sent out the cockerel, who had been given a head for this purpose, to announce that all animals should come to the square so that the King could give them a head.
The cockerel went round the whole of the village and every body who heard the message rushed to the square. 
When he thought e had informed every body, and was just making his way back to the square, the cockerel spotted the crab, without his head, meandering down a track on his way to the river bank. 
The cockerel advised him to make his way back to the square quickly, but crab just shrugged. 
“I am the secretary of the King”, he said, “the King will keep a head for me, I am sure of it. 
I need to have a quick bath. I will see you bye and bye”. And he sauntered on his way.
