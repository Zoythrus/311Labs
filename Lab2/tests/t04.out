cockel cockel cockel cockel, evy evy meanding riv saunted we 
